sklearn/gaussian_process/correlation_models.py

```diff
@@ -17,9 +17,9 @@ def absolute_exponential(theta, d):
     Absolute exponential autocorrelation model.
     (Ornstein-Uhlenbeck stochastic process)::
 
-                                            n
-        theta, dx --> r(theta, dx) = exp(  sum  - theta_i * |dx_i| )
-                                          i = 1
+                                          n
+        theta, d --> r(theta, d) = exp(  sum  - theta_i * |d_i| )
```

Comment:
To me this looks just like a mismatch between docs and code. What would have been deprecated?